CONTRIBUTING.md

```
@@ -177,26 +177,22 @@ export interface AppBenchmark {
 However, support for Astro files is still experimental. Currently, Biome only formats the frontmatter section of `.astro` files, so we use `Prettier` to format the JSX portions.
 
 ### Applying both formatters
 
-In our CI, we validate formatting using Biome only. This is intentional because Prettier’s Astro support can produce false-positive formatting errors.
-
-To ensure CI checks pass and the codebase is formatted correctly, follow these steps:
-1. Run the `prettier` script from [ensawards.org/package.json](ensawards.org/package.json) or execute the `pnpm exec prettier . --write` command from inside the `ensawards.org` package (running this from the correct location is required).
-2. Run `pnpm lint` command from the repository root.
+To ensure CI checks pass and the codebase is formatted correctly, run `pnpm lint` command from the repository root. This will run both Biome and Prettier formatting.
 
 > **NOTE (Windows users)**: After running these steps, you may see many diffs with `Contents have differences only in line separators` comment. 
```

markdownlint-cli2 (0.20.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)